File: classes/parte-11-proyecto-integrador-poliglota/172-persistencia-y-almacenamiento/implementaciones/go/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

func main() {
	line, _ := bufio.NewReader(os.Stdin).ReadString('\n')
	p := strings.Fields(line)
	almacen := map[string]string{}
	almacen[p[0]] = p[1]
	fmt.Printf("guardado=%s=%s\n", p[0], almacen[p[0]])
}

File: classes/parte-11-proyecto-integrador-poliglota/172-persistencia-y-almacenamiento/implementaciones/rust/main.rs
use std::collections::HashMap;
use std::io::Read;

fn main() {
    let mut s = String::new();
    std::io::stdin().read_to_string(&mut s).unwrap();
    let p: Vec<&str> = s.split_whitespace().collect();
    let mut almacen: HashMap<&str, &str> = HashMap::new();
    almacen.insert(p[0], p[1]);
    println!("guardado={}={}", p[0], almacen[p[0]]);
}
